packages/engine-core/src/data-proxy/DataProxyEngine.ts

```diff
@@ -48,24 +47,6 @@ export class DataProxyEngine extends Engine {
     this.remoteClientVersion = getClientVersion(this.config)
     this.headers = { Authorization: `Bearer ${apiKey}` }
     this.host = host
-
```

Comment:
We can remove this logic, prone to errors and already handled in `requestInternal`